src/openvpn/options.c

```diff
@@ -135,6 +135,9 @@ static const char usage_message[] =
     "                      udp6, tcp6-server, tcp6-client\n"
     "--proto-force p : only consider protocol p in list of connection profiles.\n"
     "                  p = udp or tcp\n"
+#if defined(TARGET_LINUX) && defined(ENABLE_MPTCP)
+    "--multipath     : Enable Multipath TCP on the TCP connections.\n"
```

Comment:
> why placing it here? Just to be next to the `--proto` option? (it would make sense but just to know, might be interesting to add this in the commit message)



> (same as below: I would recommend to use `--mptcp` to avoid confusions)
> 
> (or maybe `--proto-mptcp`?)



> or modify the behaviour of `--proto`? but then you will need to allow `mptcp{,4,6}{,-client,server}`, maybe not what we want?